### PR TITLE
doc: update example dep

### PR DESCRIPTION
**Pull request by ashenfad**
Updates the example dep to use the new repo name.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,5 +10,5 @@ Currently it offers no-op alternatives for [tqdm](https://github.com/tqdm/tqdm).
 In you `setup.cfg` or `pyproject.toml` install_requires section add:
 
 ```
-tqdm @ git+https://github.com/tignis/shenanigans.git@9f9e8e9#subdirectory=tqdm
+tqdm @ git+https://github.com/tignis/mimic.git@9f9e8e9#subdirectory=tqdm
```
Comment (orentrutner) [inline README.md:13]: Also in `tqdm/pyproject.toml`

**Review by orentrutner — COMMENTED** [review]
Proposed changes:
1. Change title from "# WARNING:" ... to "About"
2. Add a markdown note instead of the warning, e.g.
```
> [NOTE!]
> This is a public repository
```
3. Remove the exclamation marks in "This is a public repository!!!"
4. Replace "troublesome" -> "specific"

File: README.md (hunk 1 above)
Comment (ashenfad) [reply README.md:12]: Updated with 360add4

File: README.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
-# WARNING: This is a public repository!!!
+[NOTE!] This is a public repository.
```
Comment (orentrutner) [inline README.md:1]: You may have to add a larger than character before [!NOTE] and move the text to the following line, also after a larger than character. I'm just learning about it too. https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts
```
> [NOTE!]
> Some message
```

**Review by orentrutner — APPROVED** [review]

**Review by orentrutner — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ashenfad) [reply README.md]: My linter really wanted to fight me on that one but fixed in 07b06fe.

**Pull request merged**